content/en-US/api/fontVariantCaps.md

```diff
@@ -0,0 +1,102 @@
+---
+recipe: api-interface
+title: 'CanvasRenderingContext2D.fontVariantCaps'
+mdn_url: /en-US/docs/Web/API/CanvasRenderingContext2D/fontVariantCaps
+specifications: [[specURL]]#[[HeadingAnchor]]
+browser_compatibility: api.CanvasRenderingContext2D.fontVariantCaps
+---
+
+
+**When this feature ships, the content below will live on MDN under
+[developer.mozilla.org/en-US/docs/Web/CanvasRenderingContext2D/fontVariantCaps](https://developer.mozilla.org/en-US/docs/Web/CanvasRenderingContext2D/fontVariantCaps).**
+
+## Description
+
+The `fontVariantCaps` property of `CanvasRenderingContext2D` allows developers
+to control the use of alternate glyphs for capital letters. If multiple size 
+of capital letter glyphs are available for the chosen font, this 
+`CanvasRenderingContext2D` property choose the one with appropriate size; 
```

Comment:
choose -> will chose

Reply:
Also, I'd replace the ";" with a period and start a new sentence. Semicolons scare me.

Reply:
done. Thank you for even pay attention to my punctuation.

Reply:
This was a correct use of the semi-colon, but this passage can be done either way.